### PR TITLE
Features/pull censorship from katsu

### PR DESCRIPTION
**Pull request by gsfk**
First pass at getting censorship params from katsu, needs testing.

### CODE REVIEW

File: bento_beacon/config_files/config.py [hunk 1]
```diff
@@ -114,6 +114,7 @@ class Config:
     KATSU_BEACON_SEARCH = "/api/beacon_search"
     KATSU_SEARCH_OVERVIEW = "/api/search_overview"
     KATSU_PRIVATE_OVERVIEW = "/api/overview"
+    KATSU_PUBLIC_OVERVIEW = "api/public_overview"
```
Comment (davidlougheed) [inline bento_beacon/config_files/config.py:117]: inconsistent slash prefix with other variables

File: bento_beacon/app.py [hunk 2]
```diff
@@ -57,12 +59,41 @@
...
+        
+        if max_filters is not None and count_threshold is not None:
+            break
+        sleep(5 + 5*tries)
+
+    # either params retrieved or we hit max retries
+    if max_filters is None or count_threshold is None:
+        # kill container
+        raise RuntimeError("unable to retrieve censorship settings from katsu")
+    else:
```
Comment (davidlougheed) [inline bento_beacon/app.py:91]: don't need else here, since raise will always break out of the if block

**Review by davidlougheed — CHANGES_REQUESTED** [review]

File: bento_beacon/config_files/config.py (hunk 1 above)
Comment (gsfk) [reply bento_beacon/config_files/config.py:117]: huh

File: bento_beacon/app.py (hunk 2 above)
Comment (gsfk) [reply bento_beacon/app.py:91]: yes, this is here for emphasis only, perhaps the emphasis isn't needed.

File: bento_beacon/config_files/config.py (hunk 1 above)
Comment (gsfk) [reply bento_beacon/config_files/config.py]: fixed, looks like urlunsplit was covering for me here

File: bento_beacon/app.py (hunk 2 above)
Comment (gsfk) [reply bento_beacon/app.py]: removed

**Review by davidlougheed — APPROVED** [review]
lgtm

**Pull request merged**